.drone/drone.yml

```diff
@@ -122,6 +122,52 @@ steps:
 depends_on:
   - Test
 
+volumes:
+  - name: docker
+    host:
+      path: /var/run/docker.sock
+---
+kind: pipeline
+type: docker
+name: Deploy-To-Deployment-Tools 
+platform:
+  os: linux
+  arch: amd64
+trigger:
+  ref:
+    - refs/heads/main
+
+steps:
+  - name: Update Deployment Tools 
+    image: us.gcr.io/kubernetes-dev/drone/plugins/updater
+    settings:
+      config_json: |- 
+        {
+          "destination_branch": "master",
+          "pull_request_branch_prefix": "cd-agent",
+          "pull_request_enabled": true,
+          "pull_request_existing_strategy": "fail",
+          "pull_request_message": "Created by Drone build https://drone.grafana.net/api/repos/grafana/agent/builds/${DRONE_BUILD_NUMBER}\n\nBuild commit: ${DRONE_COMMIT_LINK}\n",
+          "pull_request_team_reviewers": [
+            "rfratto",
+            "mattdurham",
+            "gouthamve"
+          ],
+          "repo_name": "deployment_tools",
+          "update_jsonnet_attribute_configs": [
+            {
+              "file_path": "ksonnet/environments/platform-observability/waves/agent.libsonnet",
+              "jsonnet_key": "dev_canary",
+              "jsonnet_value": "grafana/agent:$DRONE_TAG"
+            }
+          ]
+        }
+      github_token:
+        from_secret: gh_token
+ 
+depends_on:
+  - Containerize
```

Comment:
Looking at the whole `drone.yml` plus the Makefile, I think that this won't work and you should depend instead on `Release`. If I understand correctly, the `Containerize` builds but does not push the Docker image to the registry, that is done in `Release` via `make ... publish`.

Reply:
Containerize does push the images through dockerx, release is for non-container release assets :)

Reply:
Then you are golden 👍

Reply:
Yep, I was looking at the logs and noticed that `Containerize` does push things.